sql/moz-fx-data-shared-prod/org_mozilla_fenix_derived/releases_v1/bigconfig_custom_rules.sql

```diff
@@ -0,0 +1,31 @@
+-- {
+--   "name": "Fenix releases version format",
+--   "alert_conditions": "value",
+--   "range": {
+--     "min": 0,
+--     "max": 1
+--   },
+--   "collections": ["Test"],
+--   "owner": "",
+--   "schedule": "Default Schedule - 13:00 UTC"
+-- }
+SELECT
+  ROUND((COUNTIF(NOT REGEXP_CONTAINS(version, r"^[0-9]+\..+$"))) / COUNT(*) * 100, 2) AS perc
+FROM
+  `{{ project_id }}.{{ dataset_id }}.{{ table_name }}`;
+
+-- {
+--   "name": "Fenix releases product check",
+--   "alert_conditions": "value",
+--   "range": {
+--     "min": 0,
+--     "max": 1
+--   },
+--   "collections": ["Test"],
+--   "owner": "",
+--   "schedule": "Default Schedule - 13:00 UTC"
+-- }
```

Comment:
The config for the custom SQL rules lives in these SQL comments. Not ideal, but this should hopefully be only temporary and should be easy to migrate over once BigConfig supports these custom rules